rmgpy/molecule/util.py

```diff
@@ -48,7 +48,10 @@ def retrieveElementCount(obj):
                 element, count = match.groups()
                 if count is '':
                     count = 1
-                element_count[element] = int(count)
+                if element in element_count:
```

Comment:
How did not having this section create problems in disconnected inchi generation?